eZ/Publish/Core/Persistence/Legacy/Tests/UserPreference/HandlerTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Persistence\Legacy\Tests\UserPreference;
+
+use eZ\Publish\Core\Persistence\Legacy\UserPreference\Gateway;
+use eZ\Publish\Core\Persistence\Legacy\UserPreference\Mapper;
+use eZ\Publish\Core\Persistence\Legacy\UserPreference\Handler;
+use eZ\Publish\SPI\Persistence\UserPreference\UserPreferenceSetStruct;
+use eZ\Publish\SPI\Persistence\UserPreference\UserPreference;
+use PHPUnit\Framework\TestCase;
+
+class HandlerTest extends TestCase
+{
+    const USER_PREFERENCE_ID = 1;
+
+    /** @var \eZ\Publish\Core\Persistence\Legacy\UserPreference\Gateway|\PHPUnit\Framework\MockObject\MockObject */
```

Comment:
Please use multiline PhpDoc (unless modifying some old class which already has this inconsistency)